lib/identity/hostdata/config_reader.rb

```diff
@@ -0,0 +1,106 @@
+module Identity
```

Comment:
can we add some requires here?

```suggestion
require 'logger'
require 'fileutils'
require 'yaml'

module Identity
```